var/spack/repos/builtin/packages/gmp/package.py

```diff
@@ -62,4 +72,22 @@ def configure_args(self):
         args += self.enable_or_disable("libs")
         if "libs=static" in self.spec:
             args.append("--with-pic")
+        if '+stage1' in self.spec:
+            glibc = self.spec['glibc']
+            common_flags=(" ".join([
+                '-isystem ' ,
+                glibc.prefix.include,
+                '--sysroot=/',
+                '-B ', glibc.prefix,
```

Comment:
I may be mistaken but I think white-space between `-B /path` is either not allowed or leads to issues with libtool and friends. Maybe `f"-B{prefix}"` to be on the safe side?

Reply:
Sure. FWIW, it can cause issues with libtool, but we are forced to work around libtool anyway because it likes to _silently ignore_ -B flags (with or without space) in some situations.